source/AndroidResolver/test/AndroidResolverIntegrationTestsUnityProject/Assets/ExternalDependencyManager/Editor/TestAdditionalDuplicateDependencies.template

```diff
@@ -0,0 +1,5 @@
+<dependencies>
+  <androidPackages>
+    <androidPackage spec="com.google.android.gms:play-services-base:12.0.1" />
```

Comment:
Recommend to add something with `+` like 
```
    <androidPackage spec="com.google.firebase:firebase-app-unity:5.+">
```

Reply:
Is it just the version number that you are interested in adding to the test case or do you want to swap out play-services for firebase-app-unity too?

Reply:
Add a new one so that we can test for both cases.

Reply:
Added 2 new cases to the duplicate test (a package with exact same version in a different dependencies.xml file and a package with wildcard versioning).